### PR TITLE
GA events part 2

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Part of the 665

This PR adds GA events for
- input/output currency select change
- input/output token amount change
- switch input/output tokens

Also moves set max balance event to swap events file

### CODE REVIEW

File: src/custom/utils/analytics/swapEvents.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+export function setMaxSellTokensAnalytics() {
+  _reportEvent({
+    category: Category.SWAP,
+    action: 'Set Maximun Sell Tokens',
+  })
+}
+
+export function _changeSwapAmountAnalytics(field: Field, value: number) {
```
[the remark this answers is not in the document]
Comment (nenadV91) [reply src/custom/utils/analytics/swapEvents.ts]: Good catch, fixed

**Review by W3stside — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
Approved with comments

**Comment by anxolin** [comment]
Soft approve, my comments for this PR are written in this other, since i was saying the same changes (probably the base branch is wrong) https://github.com/cowprotocol/cowswap/pull/762#pullrequestreview-1026465766

**Comment by anxolin** [comment]
Also, you need to add test instructions

**Comment by elena-zh** [comment]
Since the PR is closed, I created related to is issue #773 

Also, I'm not sure if it is an issue or not: each time when I input preceding zeros, or zeros in a decimal part, a new GA event is recorded. 
image
image


Is this OK?

**Pull request merged**